content/en/docs/Configuration/p8s-jaeger-grafana/tracing/tempo.md

```diff
@@ -40,13 +47,26 @@ spec:
   external_services:
     grafana:
       in_cluster_url: http://grafana.istio-system:3000
-      url: http://my-grafana-host
+      url: https://grafana.apps-crc.testing/
 ```
 
-We also need to set up a default [Tempo datasource](https://grafana.com/docs/grafana/latest/datasources/tempo/) in Grafana. 
+#### Set up a Tempo Datasource in Grafana
+
+We also need to set up a default [Tempo datasource](https://grafana.com/docs/grafana/latest/datasources/tempo/) in Grafana: 
 
 ![Kiali grafana_tempo](/images/documentation/configuration/grafana_tempo_ds.png)
 
+For this, we go to the Home menu in Grafana, click in _data sources_, and then in the _Add data source_. Select the Tempo data source and fill the values: 
+
+![Kiali grafana_tempo](/images/documentation/configuration/tempo_ds.png)
+
+The most important values to set up are the following: 
+
+- Mark the data source as default, so the url from Kiali redirects properly to the Tempo data source. 
+- Update the http url. This is the internal url of the http tempo frontend service. E.g. `http://tempo-tempo-query-frontend.tempo.svc.cluster.local:3200/`
+
+#### Additional configuration 
+
 The _Traces_ tab will show your traces in a bubble chart:
```

Comment:
Make it clear we have transitioned to talking about the Kiali UI (not the Grafana UI):
```suggestion
The _Traces_ tab in the Kiali UI will show your traces in a bubble chart:
```